src/pages/release-management.md

```diff
@@ -4,7 +4,7 @@ Effective management of versions and changes for Catena-X standards, KITs and op
 
 ## Support Dates
 
-![Support Dates](/img/support-dates.jpg)
+[![Support Dates](/img/support-dates.jpg)](/img/support-dates.jpg)
```

Comment:
Even if /img/... works, you should work with @site/static/img/ to indicate, that the images is really a static image and not in the current folder in an img folder